gcloud/datastore/_implicit_environ.py

```diff
@@ -126,7 +122,8 @@ def get_connection():
     :rtype: :class:`gcloud.datastore.connection.Connection`
     :returns: A connection defined with the proper credentials.
     """
-    return get_scoped_connection(Connection, SCOPE)
+    credentials = get_credentials()
```

Comment:
Allowing users to pass `scope` is not important, since these APIs have specific scopes they need. I meant something that would automatically call `get_credentials()` if the user didn't pass any.

This has been done in #864 as the `Connection.from_environment` factory.